SECP-service/src/main/resources/assets/app/js/password/password-controller.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+angular.module('SECP')
+    .controller('PasswordController', ['$scope', '$window', function ($scope, $window) {
+        $scope.sendEmail = function () {
+            swal('email sent.');    //TO DO ADD SERVER SIDE LOGIC TO SEND EMAIL
+        }
+    }])
+    .directive('compareTo', function () {
```

Comment:
why do we have a new directive. I have one already for comparing a password and confirm password. Take a look at how i did it for the register.html.